task_11/css/helpers.css

```diff
@@ -0,0 +1,254 @@
+/*style reset*/
+*{
+    box-sizing: border-box;
+    font: 100%/1.4 'Proxima Nova', Georgia, sans-serif;
+}
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+    vertical-align: baseline;
+}
+
+
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+
+/*buttons*/
+.button{
+    padding: 1em 2em;
+    text-transform: uppercase;
+    font-weight: 700;
+    letter-spacing: 1px;
+    border-radius: 3px;
+    cursor: pointer;
+    transition: all ease-out 0.2s;
+}
+
+.button:hover{
+    opacity: 0.7;
+}
+
+.button__bordered{
+    background: #fff;
+    border: 1px solid #c9d0e1;
+    color: #a5abc5;
+}
+
+.button__green{
+    background: #5fcac6;
+    border: 1px solid #54aca9;
+    color: #fff;
+}
+
+.button__empty{
+    color: #a5abc5;
+    background: none;
+    border: none;
+    cursor: pointer;
+}
+
+.button__empty:hover{
+    opacity: 0.6;
+}
+
+/*font awesome icons*/
+
+.fas{
+    padding: 0 0.5em;
+}
+
+/*custom select*/
+
+.select__custom{
+    background: #f7f7fa;
+    border: none;
+    color: #9098b7;
+    padding: 1em 4em 1em 2em;
+    text-align-last: center; /*for Chrome only*/
+    appearance:none;
+    -moz-appearance:none; 
+    -webkit-appearance:none;
+}
+
+.select-container{
+    position: relative;
+    overflow: hidden;
+    border: 1px solid #d7d8e0;
+    border-radius: 3px;
+}
+
+.select-container:after {
+    content: '▼';
+    color: #9098b8;
+    right: 15px;
+    top: 0px;
+    height: 40px;
+    padding: 18px 0px 0px 13px;
+    border-left: 1px solid #d7d8e0;
+    position: absolute;
+    pointer-events: none;
+}
+
+select::-ms-expand {
+display: none;
+}
+
+
+
+/*custom radio buttons*/
+
+.radio_custom input[type="radio"]{
+    display: none;
+}
+
+.radio_custom input[type="radio"]:checked+label{
+    border: 5px solid #76a7e1;
+    border-radius: 5px;
+}
+
+.radio_custom input[type="radio"]:checked+label{
+    border: 5px solid #76a7e1;
+    border-radius: 5px;
+    position: relative;
+}
+
+.radio_custom input[type="radio"]:checked+label:before{
+    font-family: Font Awesome\ 5 Free;
```

Comment:
Take a look at this solution for icon, maybe you will like it https://codepen.io/lempiy/pen/aEQrxg

Reply:
Cool idea with icon, i took it into project.